applications/API/errors.py

```
@@ -0,0 +1,472 @@
+                code=current_code,
+                extra={"errors": original_detail},
+            )
+        else:
+            payload = _build_payload(
+                status_code=status_code,
+                detail=str(original_detail),
+                code=current_code,
+            )
+
+    response.data = payload
+    return response
```

Pylint (4.0.3)

[convention] Final newline missing

(C0304)